Cairo Desktop/Cairo Desktop/MenuBar.xaml.cs

```diff
@@ -919,5 +903,24 @@ public void ExecuteOpenSearchResult(object sender, ExecutedRoutedEventArgs e)
             }
         }
         #endregion
+
+
+        private void Menu_DragOver(object sender, DragEventArgs e)
```

Comment:
I think we can remove this now right? This should be handled properly in StacksContainer